.devcontainer/devcontainer.json

```diff
@@ -1,16 +1,19 @@
 {
     "name": "devcontainer",
     "image": "ghcr.io/kubefirst/devcontainers/full",
+    "forwardPorts": [
+        3000
+    ],
     "features": {
-      "ghcr.io/shinepukur/devcontainer-features/vale:1": {}
+        "ghcr.io/shinepukur/devcontainer-features/vale:1": {}
     },
     "customizations": {
-      "vscode": {
-        "extensions": [
-          "unifiedjs.vscode-mdx",
-          "ChrisChinchilla.vale-vscode"
-        ],
-        "settings": {}
-      }
+        "vscode": {
```

Comment:
I guess this file hasn't been edited in a while, bunch of auto-format spacing stuff